far/changelog_eng

```diff
@@ -3,6 +3,10 @@ This file is a translation of the main russian changelog and is provided by volu
 It might not always be as up to date as the main changelog.
 =======================================================================================
 
+MZK 07.03.2020 12:45:31 -0800
+
+1. Переработал страницу справки @System.CopySecurityOptions, английскую и русскую.
```

Comment:
This probably belongs to the other changelog (and vice versa)?

Reply:
Sorry for the mess. Let's try again.